notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,921 @@
+{
```

Comment:
<p>A dataframe should be displayed using <code>display(prediction_result.head())</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>